content/docs/reference-react-dom.md

```diff
@@ -6,54 +6,55 @@ category: Reference
 permalink: docs/react-dom.html
 ---
 
-If you load React from a `<script>` tag, these top-level APIs are available on the `ReactDOM` global. If you use ES6 with npm, you can write `import ReactDOM from 'react-dom'`. If you use ES5 with npm, you can write `var ReactDOM = require('react-dom')`.
+React, `<script>` elemanıyla yüklenirse, bu üst seviye API'ları `ReactDOM` üzerinden kullanılabilirsiniz. Mesela ES6 ile npm kullanıyorsanız, `import ReactDOM from 'react-dom'` yazabilir. ES5 ile npm kullanıyorsanız `var ReactDOM = require('react-dom')` yazabilirsiniz.
 
-## Overview {#overview}
+## Genel Taslak {#overview}
 
-The `react-dom` package provides DOM-specific methods that can be used at the top level of your app and as an escape hatch to get outside of the React model if you need to. Most of your components should not need to use this module.
+`react-dom` uygulamanın en üst seviyesinde, DOM'a özel metodlar sunar ve buda gerektiğinde React model'inin dışına çıkabilmenizi sağlar. Bu arada, çoğu bileşenlerin bu modülü kullanmasına gerek olmaz.
 
 - [`render()`](#render)
 - [`hydrate()`](#hydrate)
 - [`unmountComponentAtNode()`](#unmountcomponentatnode)
 - [`findDOMNode()`](#finddomnode)
 - [`createPortal()`](#createportal)
 
-### Browser Support {#browser-support}
+### Tarayıcı desteği {#browser-support}
 
-React supports all popular browsers, including Internet Explorer 9 and above, although [some polyfills are required](/docs/javascript-environment-requirements.html) for older browsers such as IE 9 and IE 10.
 
-> Note
+React, Internet Explorer 9 ve üzeri de dahil, tüm popüler tarayıcıları desteklemekte, ama IE 9 ve IE 10 gibi eski tarayıcılarda [bazı eklentiler gerekmektedir](/docs/javascript-environment-requirements.html)  
```

Comment:
Suggestion: 
> React, Internet Explorer 9 ve üzeri de dahil olmak üzere, tüm popüler tarayıcıları desteklemektedir. Ancak, IE 9 ve IE 10 gibi eski tarayıcılar için bazı polyfill'ler (eklentiler) gerekmektedir.

The word `Polyfill` has no corresponding in Turkish. `Eklenti` can be used, but it is better if we specify the original word somehow.